AppGrid.qml

```diff
@@ -78,13 +95,30 @@ Page {
         "com.commerzbank.photoTAN": "/icons/photoTAN@4x.png"
     }
     property var labelMap: {
+        "be.ccb.safeonweb": qsTr("Safeonweb"),
+        "de.stocard.stocard": qsTr("Stocard"),
+        "s1m.savertuner": qsTr("SaverTuner"),
+        "mobi.inthepocket.bcmc.bancontact": qsTr("Bancontact"),
+        "com.symantec.mobilesecurity": qsTr("Norton 360"),
+        "com.bookmark.money": qsTr("Money Lover"),
+        "com.lscsmartconnection.smart": qsTr("LSC Smart Connect"),
+        "be.bmid.itsme": qsTr("Itsme"),
+        "be.engie.smart": qsTr("Engie Smart"),
+        "be.ixor.doccle.android": qsTr("Doccle"),
+        "com.codesynd.cashfree": qsTr("Bonsai"),
+        "com.asvcorp.aftershock": qsTr("AfterShock"),
+        "be.tweedehands.m": qsTr("2dehands"),
+        "be.argenta.bankieren": qsTr("Argenta"),
+        "be.belgacom.hello": qsTr("MyProximus"),
```

Comment:
If this apps has same names, I don't understand for why you added this code. Maybe better to remove it.